### PR TITLE
Onapsis Repackaging 

### PR DESCRIPTION
**Pull request by v-prasadboke**
Onapsis Platform repackaging and link update for data connector

### CODE REVIEW

**Comment by v-sabiraj** [comment]
@v-prasadboke, please update text "Azure Sentinel" to "Microsoft Sentinel" in Solutions/Onapsis Platform/Data Connectors/OnapsisPlatform.json and maintemplate. Thanks.

**Comment by v-mchatla** [comment]
Hi @v-prasadboke,
Please undo the changes done for the creatUI and repackage the solution as we should not update any description as part of repackaging.
Thanks

File: Solutions/Onapsis Platform/Package/createUiDefinition.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
             "name": "dataconnectors1-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "This solution installs the data connector to ingest alarms triggered in the Onapsis Platform in Common Event Format (CEF) into Microsoft Sentinel. After installing the solution, configure and enable this data connector by following guidance in Manage solution view."
+              "text": "The solution installs the data connector to ingest alarms triggered in the Onapsis Platform in Common Event Format (CEF) into Microsoft Sentinel. After installing the solution, configure and enable this data connector by following guidance in Manage solution view."
```
Comment (v-mchatla) [inline Solutions/Onapsis Platform/Package/createUiDefinition.json:63]: Discard this change as well and update the zip file.

**Review by v-mchatla — APPROVED** [review]
Added short link for the parser. Changes look good.

**Pull request merged**